Jenkinsfile

```diff
@@ -1,49 +1,16 @@
+@Library('defra-library@0.0.2')
+import uk.gov.defra.ffc.DefraUtils
+def defraUtils = new DefraUtils()
+
 def registry = '562955126301.dkr.ecr.eu-west-2.amazonaws.com'
 def regCredsId = 'ecr:eu-west-2:ecr-user'
 def kubeCredsId = 'awskubeconfig002'
-def ingressServer = "ffc.aws-int.defra.cloud"
 def imageName = 'ffc-demo-calculation-service'
 def repoName = 'ffc-demo-calculation-service'
-def repoUrl = ''
-def commitSha = ''
-def branch = ''
 def pr = ''
 def mergedPrNo = ''
 def containerTag = ''
 
-def getMergedPrNo() {
-    def mergedPrNo = sh(returnStdout: true, script: "git log --pretty=oneline --abbrev-commit -1 | sed -n 's/.*(#\\([0-9]\\+\\)).*/\\1/p'").trim()
-    return mergedPrNo ? "pr$mergedPrNo" : ''
-}
-
-def getRepoURL() {
-  return sh(returnStdout: true, script: "git config --get remote.origin.url").trim()
-}
-
-def getCommitSha() {
-  return sh(returnStdout: true, script: "git rev-parse HEAD").trim()
-}
-
-def getVariables(repoName) {
-    def branch = BRANCH_NAME
-    // use the git API to get the open PR for a branch
-    // Note: This will cause issues if one branch has two open PRs
-    def pr = sh(returnStdout: true, script: "curl https://api.github.com/repos/DEFRA/$repoName/pulls?state=open | jq '.[] | select(.head.ref == \"$branch\") | .number'").trim()
-    def rawTag = pr == '' ? branch : "pr$pr"
-    def containerTag = rawTag.replaceAll(/[^a-zA-Z0-9]/, '-').toLowerCase()
-    return [branch, pr, containerTag,  getMergedPrNo(), getRepoURL(), getCommitSha()]
-}
-
-def updateGithubCommitStatus(message, state, repoUrl, commitSha) {
-  step([
-    $class: 'GitHubCommitStatusSetter',
-    reposSource: [$class: "ManuallyEnteredRepositorySource", url: repoUrl],
-    commitShaSource: [$class: "ManuallyEnteredShaSource", sha: commitSha],
-    errorHandlers: [[$class: 'ShallowAnyErrorHandler']],
-    statusResultSource: [ $class: "ConditionalStatusResultSource", results: [[$class: "AnyBuildResult", message: message, state: state]] ]
-  ])
-}
-
 def buildTestImage(name, suffix) {
   sh 'docker image prune -f || echo could not prune images'
```

Comment:
This will need updating to use the `buildTestImage()` function from the shared Jenkins library once we have agreed and standardised how we structure docker-compose files for running tests across the FFC demo repositories.